.scalafmt.conf

```diff
@@ -0,0 +1,24 @@
+version = 3.7.17
```

Comment:
This will reformat the code on save into nice formats.  You can ignore this from your review for now....sorry for mixing concerns.

Reply:
Got it, sounds good